src/app/pages/pages.styles.tsx

```diff
@@ -34,4 +34,5 @@ export const popupPageContent = css`
 export const popupPageTitle = css`
   font-size: 32px;
   line-height: 44px;
+  max-width: 264px;
```

Comment:
Could this go on the component itself?

Reply:
We need to have a larger discussion about CSS hardcoded values and units. These one-of values are going to make our life really hard when making design changes. Why do we have units on line-height?

Reply:
I guess it is better in the end, like you pointed out, the view is based on a url ...we want it to be based on the breakpoints instead.